index.html

```diff
@@ -264,350 +274,353 @@ <h2>Intro Section of the Document</h2>
+            <li><span class="rfc2119-assertion" id="submission-requirements-summary-document-access-usage">It MUST contain Access/Usage restrictions about the binding, protocol, implementation, etc., using the terminology and/or documents of the submitter.</span> A non-exhaustive list of examples of restrictions:
+              <p class="ednote">[DP] the list is not correctly nested but was already that way in the <a href="https://github.com/w3c/wot-bindings-registry/blob/72f86e706a7292b1419e4c088fb8e1c16e1a4e01/requirements.md#req-summ">original MD document</a>.</p>
+            </li>
+            <li>Reading the binding document</li>
+            <li>Reading the protocol specification</li>
+            <li>Implementing a non-commercial device/Thing</li>
+            <li>Implementing a non-commercial Consumer application/driver</li>
+            <li>Conditions for commercial use, e.g., building a commercial product with the binding</li>
+            <li>Making a statement about your product&#39;s supporting that binding</li>
+            <li><span class="rfc2119-assertion" id="submission-requirements-summary-document-dependency">If the entry depends on another one, it MUST specify the exact version of the dependency upon which it depends at the time of submission.</span></li>
+            <li><span class="rfc2119-assertion" id="submission-requirements-summary-document-availability">The availability of the machine-readable documents MUST be indicated in the summary document using the submission mechanism.</span> Also see <a href="#req-docs">Req-Docs</a>.</li>
+            <li><span class="rfc2119-assertion" id="submission-requirements-summary-document-previous-version">The previous version of the summary document MUST be listed as a link</span>.</li>
+            <li><span class="rfc2119-assertion" id="submission-requirements-summary-document-chronological-ordering">If the chronological ordering of the entries is not clear from the version string, the summary document MUST explain the ordering mechanism.</span></li>
+          </ul>
+        </section>
+        
+        <section id="req-trancurr">
+          <h3>Transition</h3>
+          <p>Transition from <code>Initial</code> to <code>Current</code></p>
+          <ul>
+            <li>Starting from the initial submission, each binding has to demonstrate a certain level of concrete development maturity. This process involves real-world testing, which can take place in Plugfests, independent testing events, or even informal collaboration between developers. These testing events do not have to be organized by W3C and can be conducted remotely, including over VPN. The goal is to demonstrate that the binding correctly maps protocol operations and is well understood by at least two parties.</li>
```

Comment:
The initial wording can be made assertions

Reply:
done so in https://github.com/w3c/wot-bindings-registry/pull/17/commits/12a04f8d694e77865e763fbc5e1a1e29ee711214